dev-utils/radio_crawler/fetch_xiph.py

```diff
@@ -22,7 +22,7 @@
 XIPH_CACHE = "xiph.pickle"
 
 
-class XiphPlaylist(object):
+class XiphPlaylist():
```

Comment:
```suggestion
class XiphPlaylist:
```

Reply:
Thanks @declension I have resolved this.